src/components/font_size_editor/font_size_editor.xml

```diff
@@ -1,24 +1,31 @@
 <templates>
   <t t-name="o-spreadsheet-FontSizeEditor" owl="1">
-    <div class="o-dropdown o-font-size-editor" t-ref="FontSizeEditor">
-      <input
-        type="number"
-        min="1"
-        max="400"
-        class="o-font-size bg-transparent border-0 ps-1"
-        t-on-keydown="onInputKeydown"
-        t-on-click.stop=""
-        t-on-focus.stop="onInputFocused"
-        t-att-value="currentFontSize"
-        t-on-change="setSizeFromInput"
-        t-ref="inputFontSize"
-      />
-      <div title="Font Size" t-on-click="() => this.toggleFontList()">
-        <t t-call="o-spreadsheet-Icon.TRIANGLE_DOWN"/>
+    <div class="o-dropdown" t-ref="FontSizeEditor">
+      <div
+        class=" o-font-size-editor h-100 d-flex align-items-center"
+        t-att-class="props.class"
+        title="Font Size"
+        t-on-click="() => this.toggleFontList()">
```

Comment:
```suggestion
        t-on-click="this.toggleFontList">
```

Reply:
Does it change anything in practice ? We use the arrow function version in 95% of the code base, is it worth changing ?

Reply:
no it doesn't really change anything in practice. Just an additional anonymous function in tracebacks when debugging.
It's a minor detail